### DISCUSSION TITLE
Tagging Directories

### DISCUSSION THREAD

**Discussion by Your-Name-Here** [TagStudioDev/TagStudio#487] [Ideas]
I have finally downloaded TagStudio recently. As a developer, my main interest was to be able to tag whole directories to be able to find projects that I've started and stopped. If this is already possible, feel free to get rid of this discussion. 

Also, I added 'Desktop' as a library... lol omg... we should be able to have something like a 
``.tagstudioignore`` for 
``global: */node_modules/`` 
and other library code directories.

**Comment by CyanVoxel** [comment]
Both of these are planned:
Taggable directories: #74
`gitignore`-like filtering: #14

If you would like to generate and apply tags based on their current folders (one time operation, does not sync with folder contents) then see the "Folders to Tags" "feature in "Macros" menu. I would also recommend exploring the "Manage File Extensions" list in the "Edit" menu to help with file filtering in the meantime.

**Comment by Your-Name-Here** [comment]
Thank you for the super quick reply and sorry for not checking thoroughly enough.

**Comment by a84r7a3rga76fg** [comment]
I need this so badly. If this becomes a thing, please make it so the content in the directory doesn't appear in the result menu until we click on the directory (same as how file explorer works).
